modules/apps/app-builder/app-builder-api/src/main/java/com/liferay/app/builder/constants/AppBuilderWebKeys.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.app.builder.web.internal.constants;
+package com.liferay.app.builder.constants;
```

Comment:
Hey @naraandrade , this change requires a version increment. Please run `gw baseline`. Thx

Reply:
I'll do it. Thank you :)